.github/scripts/check_labels.py

```diff
@@ -1,63 +1,19 @@
 #!/usr/bin/env python3
-"""check_labels.py"""
 
-from typing import Any, List
-
-from export_pytorch_labels import get_pytorch_labels
+from typing import Any
 from gitutils import (
     get_git_remote_name,
     get_git_repo_dir,
     GitRepo,
 )
 from trymerge import (
-    _fetch_url,
-    gh_post_pr_comment,
+    LABEL_ERR_MSG,
+    has_required_labels,
+    add_label_err_comment,
+    delete_all_label_err_comments,
```

Comment:
How about we move these to some new common library as well?

Reply:
As discussed offline, this would ideally require extracting `GitHubPR` class and all its dependencies from `trymerge` to common module(s) as well, to avoid circular dependencies and improve repo structure. Given it'd be a slightly larger refactoring job, happy to do it as a separate PR after this one is merged.